src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/DependencyAnalysis/LLVMMethodCodeNode.cs

```diff
@@ -116,39 +122,24 @@ public void InitializeDebugInfo(MethodDebugInformation debugInfo)
         public void InitializeLocalTypes(TypeDesc[] localTypes)
         {
         }
-    }
-
-    internal sealed class LlvmMethodBodyNode : LLVMMethodCodeNode, IMethodBodyNode
```

Comment:
I deleted the `LlvmMethodBodyNode/LLVMMethodCodeNode` fork, it did not look to serve any meaningful purpose. `MethodCodeNode` does not use such pattern either.